ginga/rv/plugins/RC.py

```diff
@@ -346,5 +346,23 @@ def ginga(self, method_name, *args, **kwdargs):
         _method = getattr(self.fv, method_name)
         return self.fv.gui_call(_method, *args, **kwdargs)
 
+    def canvas(self, chname, command, *args, **kwargs):
+        chinfo = self.fv.get_channel(chname)
+        canvas = chinfo.viewer.get_canvas()
+        if command == 'add':
+            klass = canvas.get_draw_class(args[0])
+            newargs = args[1:]
+            obj = klass(*newargs, **kwargs)
+            return self.fv.gui_call(canvas.add, obj)
+        elif command == 'clear':
+            canvas.delete_objects()
```

Comment:
I think we need an "else" clause here, probably raising an error if the command is not one of the recognized ones.